logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -391,6 +399,7 @@ internal class MessageSenderImpl internal constructor(
         action: String, // Send or Broadcast
         messageLogString: String,
         conversationId: ConversationId?,
+        retryCount: Int = 0,
```

Comment:
Suggestion: Just a matter of code uniformity for the same concept, what about using the inverse logic like [here called remainingAttempts](https://github.com/wireapp/kalium/blob/d911835d838056c4de0cbbfa99cdda4b6b93527c/logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt#L575-L576)